docker/data-science/notebooks/ch10/1.setup-the-movie-db.ipynb

```diff
@@ -80,7 +80,7 @@
     {
      "data": {
       "text/plain": [
-       "{'responseHeader': {'rf': 1, 'status': 0, 'QTime': 6723}}"
+       "{'responseHeader': {'rf': 1, 'status': 0, 'QTime': 5610}}"
```

Comment:
It looks like `engine.add_documents` is returning the Solr header, which is being printed here (and is not particularly useful).

I'd recommend that instead engine.add_documents returns a Boolean (indicating success / failure) and then IF we want to print success it can just check the boolean and print a "Success" message. I think this pattern is followed i in few places. Whatever pattern you want to use, though, I don't think we should leak the Solr-specific response header out of the `add_documents` method.